src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/GoToImplementation.test.ts

```diff
@@ -53,35 +59,6 @@ suite('Implementation', () => {
         assert.equal(implementation.range.start.line, 2);
     });
 
-    test('Implementation background file works', async () => {
```

Comment:
@ryanbrandenburg, this was consistently failing. Is this a realistic scenario? When will someone invoke Go to implementation on a background document?

Reply:
You're right, this is a silly test. I wonder how it passed on my machine even...